swim/memberlist.go

```diff
@@ -336,11 +336,12 @@ func (m *memberlist) Update(changes []Change) (applied []Change) {
 		// if change is local override, reassert member is alive
 		if member.localOverride(m.node.Address(), change) {
 			m.node.emit(RefuteUpdateEvent{})
+			newIncNo := nowInMillis(m.node.clock)
 			overrideChange := Change{
```

Comment:
I think I prefer the [old node version][1] where the change was taken as-is and only the interesting fields got overwritten. If you agree with that, can we do the same here?

I just checked Go documentation, and couldn't find a way to have mixed typespecs, like in protobuffs*. Maybe there is a way that I don't know?

[1]: https://github.com/uber/ringpop-node/pull/282#discussion_r66072017
[2]: https://developers.google.com/protocol-buffers/docs/proto
[*]: From [their docs][2]: _However, the unknown fields are not discarded, and if the message is later serialized, the unknown fields are serialized along with it – so if the message is passed on to new code, the new fields are still available._